test/setup/outputs.tf

```diff
@@ -76,6 +76,10 @@ output "domains_to_allow" {
   value = tolist([var.domain_to_allow])
 }
 
+output "essential_contacts_domains_to_allow" {
+  value = tolist(["@${var.domain_to_allow}"])
```

Comment:
Don't we have a separate var.essential_contacts_domains_to_allow?

Reply:
This is just a matter of making the build tests from the pipeline simpler to be configured. We do kind of the samething for the google groups by providing the same variable "group_email" for all of them.

Reply:
Ah, I thought this was in the 1-org output, thanks for clarifying.